lib/api.go

```diff
@@ -115,6 +115,10 @@ const (
 	AEFeeds = APIEndpoint("/feeds")
 	// AEPreview fetches a dataset preview from the registry
 	AEPreview = APIEndpoint("/preview")
+	// AERemoteList fetchs a list of the users dataset published on a given remote
+	AERemoteList = APIEndpoint("/remote/list")
+	// AERemoteRemove removes a dataset from a given remote
+	AERemoteRemove = APIEndpoint("/remote/remote")
```

Comment:
`/remote/remote` should be `/remote/remove`

Reply:
THANK YOU